### PR TITLE
Remove Nazrin::Struct and move behaviour to DataAccessor

### PR DESCRIPTION
**Pull request by AMHOL**

### CODE REVIEW

File: lib/nazrin/data_accessor/struct.rb [hunk 1]
```diff
@@ -1,6 +1,60 @@
...
+            end
+          end
+        end
+
+        def field_types
+          return @field_types if defined?(@field_types)
+          raise(
+            MissingDomainNameConfigError,
+            "No domain_name configured for #{clazz.name}"
+          ) unless config.domain_name
```
Comment (tsuwatch) [inline lib/nazrin/data_accessor/struct.rb]: Unnecessary error handling.

Comment (AMHOL) [reply lib/nazrin/data_accessor/struct.rb]: @tsuwatch Updated

**Comment by tsuwatch** [comment]
Thank you so much!

**Comment by AMHOL** [comment]
Thank you too :smile:

**Pull request merged**